package.json

```diff
@@ -43,7 +43,7 @@
     "tsconfig-paths": "^4.2.0",
     "typescript": "^5.2.2"
   },
-  "dependencies": {
+  "peerDependencies": {
     "@forlagshuset/audit-logging": "^1.0.2",
```

Comment:
@wrola actually this should remain as a hard dependency - this library actually depends on the audit-logging package and is abstracting its api (in case of higher level parts, like services etc)